substrate/frame/child-bounties/src/lib.rs

```diff
@@ -206,9 +220,25 @@ pub mod pallet {
 	>;
 
 	/// The description of each child-bounty.
+	///
+	/// This item replaces the `ChildBountyDescriptions` storage item from the V0 storage version.
 	#[pallet::storage]
-	pub type ChildBountyDescriptions<T: Config> =
-		StorageMap<_, Twox64Concat, BountyIndex, BoundedVec<u8, T::MaximumReasonLength>>;
+	pub type ChildBountyDescriptionsV1<T: Config> = StorageDoubleMap<
+		_,
+		Twox64Concat,
+		BountyIndex,
+		Twox64Concat,
+		BountyIndex,
+		BoundedVec<u8, T::MaximumReasonLength>,
+	>;
+
+	/// The mapping of the child bounty ids from storage version `V0` to the new `V1` version.
+	///
+	/// The `V0` ids based on total child bounty count [`ChildBountyCount`]`. The `V1` version ids
```

Comment:
Can you please add a note that this is just convenience and not used in the code itself?